addons/website/controllers/form.py

```diff
@@ -279,6 +278,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            warning_icon = ""
+            authenticate_message = False
+            if request.session.uid:
+                model_record = request.env[model_name]
+                user_email = request.env.user.email
+                form_email = values[model_record._mail_get_primary_email_field()]
+                if user_email != form_email:
+                    authenticate_message = _("This %(model_name)s was submitted by %(user_name)s (%(user_email)s) on behalf of %(form_email)s",
+                        model_name=model.name, user_name=request.env.user.name, user_email=user_email, form_email=form_email)
+            else:
+                warning_icon = "/!\\ "
+                authenticate_message = _("EXTERNAL SUBMISSION - Customer not verified")
+            if authenticate_message:
+                record._message_log(
+                    body=Markup('<div class="alert alert-info" role="alert">{warning_icon}{message}</div>').format(warning_icon=warning_icon, message=authenticate_message),
```

Comment:
Hello @odoo/rd-security 

Can we have an override? 🙂 

We use Markup function with content generated inside that method.
(I am not sure if we have to escape the `form_email` (used in L290) to be sure)

Thanks in advance! 🙂